rust/Cargo.toml

```diff
@@ -14,10 +14,9 @@ opentelemetry = { git = "https://github.com/open-telemetry/opentelemetry-rust",
   "trace",
 ], rev = "67bc4f9" }
 opentelemetry-zipkin = "0.17.0"
-opentelemetry-stdout = { git = "https://github.com/open-telemetry/opentelemetry-rust", features = [
```

Comment:
Can we get rid of `opentelemetry` too?

Reply:
Yea I'm doing a quick pass over this stuff and other imports. Looking good so far though, @wwkeyboard .